client/src/components/BoulderCard.jsx

```diff
@@ -141,6 +140,9 @@ const Level = styled.p`
   grid-row: 2/3;
 `;
 const Hold = styled.p`
+  text-overflow: ellipsis;
```

Comment:
I think you could also work with a styled component that contains the new stylings you added to every styled p element and style these styled components based on this new styled component. As this sentence might be confusing I will add a example:

```
const Text = styled.p`
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
`

const Hold = styled(Text)`
  grid-column: 3/5;
  grid-row: 2/3;
`
```

You could use `Text` (not sure about the name though) also for `Sector`, `Setter` and `Weighting` and your code would be less repetitive. :)

Reply:
Wow! This is soo cool!